tests/unittests/caffe2ImporterTest.cpp

```diff
@@ -1341,3 +1341,41 @@ TEST(caffe2, sparseToDense) {
   // Graph has three inputs and one output.
   EXPECT_EQ(mod.getPlaceholders().size(), 4);
 }
+
+/// Test loading NCHW2NHWC op.
+TEST(caffe2, testNCHW2NHWC) {
+  ExecutionEngine EE{BackendKind::Interpreter};
+  auto &mod = EE.getModule();
+  Function *F = mod.createFunction("main");
+
+  std::string NetDescFilename(
+      "tests/models/caffe2Models/NCHW2NHWC_predict_net.pbtxt");
+  std::string NetWeightFilename(
+      "tests/models/caffe2Models/empty_init_net.pbtxt");
+
+  Placeholder *output;
+  Context ctx;
```

Comment:
Why do we need the context?

Reply:
It is actually for getting the output shape -- although the check is just added...